packages/@divvi/mobile/package.json

```diff
@@ -124,6 +124,7 @@
     "@divvi/referral-sdk": "^2.2.0",
     "@fiatconnect/fiatconnect-sdk": "^0.5.74",
     "@fiatconnect/fiatconnect-types": "^13.3.11",
+    "@google-cloud/recaptcha-enterprise-react-native": "^18.8.0",
```

Comment:
This should be a `peerDependency` since it's a native dep.

Also we should add it to the root package.json resolutions like we did with the other native deps, to ensure only 1 version is used in the monorepo.